exp/configuration_bibo.py

```
@@ -0,0 +1,34 @@
+
+class BiBoConfig(_StableBiBoConfig):
+    """BiBo configuration for the experimental Attention Residuals model.
+
+    ``attn_res_block_size`` is measured in complete transformer decoder layers,
+    matching the official Kimi K3 Hugging Face implementation. K3 uses 12.
+    Passing ``None`` disables AttnRes and provides a standard-residual control.
+    """
+
+    model_type = "bibo_attn_res"
+
+    def __init__(self, attn_res_block_size=12, **kwargs):
```

Ruff (0.16.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)